courses/machine_learning/intro_to_tensorflow/4_keras_functional_api.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>I'd put the model compilation into the build_model function referenced above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/576/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>